lib/client/js/ui/controls/Notifications.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * The UI Control that manages the Notifications section of
+ * the BeansContent
+ *
+ * @class
+ */
+var Notifications = (function () {
+    /**
+     * @alias Notifications
+     */
+    function Notifications() {
+        // turn the arguments of this constructor into an array
+        var args = Array.prototype.slice.call(arguments);
+
+        /*
+         * @see {@link http://tobyho.com/2010/11/22/javascript-constructors-and/}
+         */
+
+        // Apply the Control constructor to the Notifications instance with the input arguments
+        Control.apply(this, args);
+    }
```

Comment:
There's a difference between a function declaration, `function foo() {}`, and an assignment of a variable to a function, `var foo = function () {};`. Function declarations should not have a semicolon. The first answer [here](http://stackoverflow.com/questions/1834642/why-should-i-use-a-semicolon-after-every-function-in-javascript) provides a good explanation as to why function declarations don't need them/don't get them.